README.rst

```diff
@@ -1,12 +1,18 @@
+##################
+frontend-app-authn
+##################
+
 |Build Status| |ci-badge| |Codecov| |semantic-release|
 
-frontend-app-authn
-====================
+********
+Purpose
+********
 
 Please tag **@openedx/vanguards** on any PRs or issues.  Thanks!
 
-Introduction
-------------
+***************
+Getting Started
+***************
 
 This is a micro-frontend application responsible for the login, registration and password reset functionality.
```

Comment:
Remove this section from here:

This is a micro-frontend application responsible for the login, registration and password reset functionality.

**What is the domain of this MFE?**
- Register page
- Login page
- Forgot password page
- Reset password page

- Progressive profiling page